Library/Formula/libdistance.rb

```diff
@@ -0,0 +1,21 @@
+class Libdistance < Formula
+  homepage "http://monkey.org/~jose/software/libdistance/"
+  url "http://monkey.org/~jose/software/libdistance/libdistance-0.2.2.tar.gz"
+  sha1 "d516e5f24ae55bcbbf93a4d14c6a53914e5b3a0a"
+
+  def install
+    # The build system is incredibly simple.
+    system "make"
+    system "make -C test"
+
+    include.install "distance.h"
+    lib.install "libdistance.a"
+
+    mkdir "#{prefix}/test"
+    cp "test/test", "#{prefix}/test/"
```

Comment:
Hmmm, actually, changing `cp` to `share.install` broke the build in an unexpected way:

```
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:519:in `rename'
Errno::ENOTDIR: Not a directory - (/usr/local/Cellar/libdistance/0.2.2/test/, /usr/local/Cellar/libdistance/0.2.2/share/test)
```

Any ideas what this is all about?